cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/scheduler_plugin_user/99-parallelcluster-scheduler-plugin.erb

```diff
@@ -1 +1,3 @@
 <%= node['cluster']['scheduler_plugin']['user'] %> ALL = (root) NOPASSWD: ALL
```

Comment:
This file only gets created if GrantsSudoPrivileges=True while I assume that we always want to give `cluster_admin_user` such permissions. Either we move the conditional logic in this file or we create a separate one.